task1/src/main/java/com/fundacionjala/movies/Movie.java

```diff
@@ -0,0 +1,32 @@
+package com.fundacionjala.movies;
+
+public  class Movie {
+
+    public static final int CHILDRENS = 2;
```

Comment:
Is it necessary to use CHILDRENS, REGULAR, NEW_RELEASE variables ?
because now you are using inherintance